### PR TITLE
Complete set of pre-defined prompts

### PR DESCRIPTION
**Pull request by TuanaCelik**
@ZanSara and @vblagoje - here are all our prompts. 
- What is the 'versioning' strategy do you think? 
- Are we ok to have the authors for `deepset/xyz` prompts be 'deepset'? Or do we envision personal contributions from deepset folk where they would like to add their name?

### CODE REVIEW

**Comment by TuanaCelik** [comment]
Fixes #12

File: prompts/deepset-question-generation copy.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+name: deepset/summarization
```
Comment (vblagoje) [inline prompts/deepset-question-generation copy.yaml:1]: Some leftover, not properly named YAML file I suppose.

**Comment by vblagoje** [comment]
Looks good @TuanaCelik ; I found one strange file. Let's see what's that about.

File: prompts/deepset-question-generation copy.yaml (hunk 1 above)
Comment (TuanaCelik) [reply prompts/deepset-question-generation copy.yaml]: Woops! Sorry, merged the fix

**Review by vblagoje — APPROVED** [review]
LGTM, let's just load these and try them out :-)

**Pull request merged**